table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -219,6 +219,7 @@ public UdbTable getTable(String tableName) throws IOException {
           .setTableName(tableName)
           .addAllDataCols(HiveUtils.toProto(table.getSd().getCols()))
           .setStorage(HiveUtils.toProto(table.getSd(), pathTranslator))
+          .putAllParameters(table.getParameters())
```

Comment:
This was missing from before? This is not related to the serde confusion, right? Is there any unittest that can cover this type of issue?

Reply:
this is probably best tested using integration test / end to end test. Mocking a hive metaserver to test this has always had the problem of not being very accurate..